policies/wasm/policies/namespace_discovery_relationship_policy/namespace_discovery_relationship_policy.rego

```diff
@@ -1,62 +1,54 @@
 # https://play.openpolicyagent.org/p/wFNhyGsIej
 package hierarchical_policy
 
-available_namespaces[namespace_key] {
-  service := input.services[_];
-  namespace_key = service.namespace
+import rego.v1
+
+available_namespaces[namespace_key] if {
```

Comment:
While this is valid Rego v1, it changes the semantics from the previous version. 

The Rego v1 equivalence of this:
```rego
available_namespaces[namespace_key] {
```
Would be this:
```rego
available_namespaces contains namespace_key if {
```
And what you've got here, i.e this:
```rego
available_namespaces[namespace_key] if {
```
would in Rego v1 mean "a map with a `namespace_key` and an implicit value of `true`. 

A good idea is to use `opa fmt --rego-v1` to have the file automatically formatted for Rego v1 compliance, and then start linting with Regal from there.